### PR TITLE
 Fix missing prefix 

### PR DESCRIPTION
**Pull request by kenzouno1**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "cmgmyr/messenger",
+    "name": "kenzouno1/messenger",
```
Comment (a-komarev) [inline composer.json]: This is fun ^_^

Comment (cmgmyr) [reply composer.json]: can you fix this please @kenzouno1?

File: src/Models/Message.php [hunk 2]
```diff
@@ -104,12 +104,12 @@ public function recipients()
      */
     public function scopeUnreadForUser(Builder $query, $userId)
     {
-        return $query->has('thread')
+          return $query->has('thread')
```
Comment (cmgmyr) [inline src/Models/Message.php]: can you run `vendor/bin/php-cs-fixer fix .`?

File: composer.json (hunk 1 above)
Comment (kenzouno1) [reply composer.json]: i'm so sorry.Because I needed to send the corrected version to my team yesterday so I sent a fork to packagist.org

**Comment by kenzouno1** [comment]
I did not think that commit after pulling was included here :(

**Comment by cmgmyr** [comment]
Thanks @kenzouno1!

**Pull request merged**